resources/markdown/who_audit_l2.md

```diff
@@ -0,0 +1,282 @@
+     Female
+     OR Male > 65 years old
+     OR Sex at birth is unknown
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Continue to display (WHO) AUDIT-C screening instructions
+- Continue to display U.S. standard drink size graphic
+- Display Question #2 and Question #3 of the (WHO) AUDIT-C (Question #3 = How often do you have four or more drinks on one occasion?)
+- Record and display the response to Question #2  and Question #3
+- Calculate (WHO) AUDIT-C score
+- Record and display (WHO) AUDIT-C score
+</pre>
+
+----------------------------------------------------------------------
+## LOGIC PATH #2: Continue screening for men <= 65:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'YES'
+AND NOT most recent (WHO) AUDIT-C Question #1 response is 'Never'
+AND Male <= 65 years old
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Continue to display (WHO) AUDIT-C screening instructions
+- Continue to display standard drink size graphic
+- Display Question #2  and Question #3 of the (WHO) AUDIT-C (Question #3 = How often do you have five or more drinks on one occasion?)
+- Record and display responses to Question #2 and Question #3
+- Calculate (WHO) AUDIT-C score
+- Record and display (WHO) AUDIT-C score
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 4: Consider presenting Question #9 and Question #10 of the (WHO) AUDIT when not administering the full (WHO) AUDIT: 
+
+## LOGIC PATH #1: Jump from Question #1 to Question #9 & Question #10 OR jump from end of (WHO) AUDIT-C to Question #9 & Question #10:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'YES'
+AND
+     Most recent (WHO) AUDIT-C Question #1 response is 'Never'
+OR
+     Most recent (WHO) AUDIT-C Question #1 score < 3
+     AND most recent (WHO) AUDIT-C Question #2 score = 0
+     AND most recent (WHO) AUDIT-C Question #3 score = 0
+     AND
+           Female
+          OR Sex at birth is unknown
+          Male > 65 years old 
+ OR   
+      Most recent (WHO) AUDIT-C Question #1 score < 4
+      AND most recent (WHO) AUDIT-C Question #2 score = 0
+      AND most recent (WHO) AUDIT-C Question #3 score = 0
+      AND Male
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Display Question #9 & Question #10 of the (WHO) AUDIT
+- Record and display responses to Question #9 & Question #10  of the (WHO) AUDIT
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 5: Consider additional screening: 
+
+## LOGIC PATH #1: Provide full (WHO) AUDIT:
+
+### Inclusion:
```

Comment:
SS: I don't think that Dave's suggestion applies here. We ALWAYS want to default to providing the full AUDIT if the score is >=3/4. The only time that we would care about providing Q9-Q10 to the patient is if their AUDIT-C is <3/4 (in which case we check to see if Q2+Q3=0. I think this logic is correct as stated.

Reply:
GM: what did we decide about males that are >=65? The logic flow indicates they would be included in the first calculation, for AUDIT-C >=3, but this says all males are created equal. 
SS: Updated to include a man's age